index.html

```diff
@@ -36,29 +36,130 @@
+        <button class="tablinks" onclick="openTab(event, 'Contact')">Contact</button>
+    </div>
     
-    <div id="mc_embed_signup">
-        <form action="https://quanteco.us5.list-manage.com/subscribe/post?u=84b372b9598d22040af9583e0&amp;id=0def27f7e4" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank">
-            
-            <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-                <input class="mdl-textfield__input" type="email" value="" name="EMAIL" id="mce-EMAIL" data-required>
-                <label class="mdl-textfield__label" for="mce-EMAIL">Email address</label>
-            </div>
+    <div id="About" class="tabcontent">
+        <p> It's time to quantify our carbon footprints.</p>
+        <p>Sign up to our mailing list to follow the launch of our app, Foodprint.</p>
+        <div id="mc_embed_signup">
+            <form action="https://quanteco.us5.list-manage.com/subscribe/post?u=84b372b9598d22040af9583e0&amp;id=0def27f7e4" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank">
+                
+                <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+                    <input class="mdl-textfield__input" type="email" value="" name="EMAIL" id="mce-EMAIL" data-required>
+                    <label class="mdl-textfield__label" for="mce-EMAIL">Email address</label>
+                </div>
 
-            <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value=""></div>
+                <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value=""></div>
 
-            <button type="submit" name="subscribe" id="mc-embedded-subscribe" class="mdl-button mdl-button--raised mdl-button--primary">
-                Subscribe
-            </button>
+                <button type="submit" name="subscribe" id="mc-embedded-subscribe" class="mdl-button mdl-button--raised mdl-button--primary">
+                    Subscribe
+                </button>
+                </div>
+            </form>
+        </div>
+    
+    </div>
+
+    <div id="Foodprint" class="tabcontent">
+     <!--    <h3>Foodprint</h3>
+    
+                width="175"
+                height="296"
+    
+    -->
+        <p> 
+            Coming soon... A recipe and diet app with ecology and health at its heart.
+        </p>
+        <p>    
+            Find the footprint behind your plate by searching the nutrition and ecological impacts 
```

Comment:
"ecological/environmental footprint"? The rest of sentence continues as if the nutritional aspect actually matters as far as the footprint goes